services/order.py

```diff
@@ -0,0 +1,35 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, MovieSession
+from django.db import transaction
+from django.contrib.auth import get_user_model

```

Comment:
Add a blank line between different import groups and order them correctly to follow PEP-8